### PR TITLE
Html css popup task

### PR DESCRIPTION
**Pull request by mimoauto**
### HTML CSS PopUp

[Demo](https://mimoauto.github.io/popup-example) |
[Code base](https://github.com/mimoauto/popup-example)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/mimoauto/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+            <input
+              class="menu-nav-check-input"
+              id="popup-active"
+              type="checkbox"
+            />
+            <label for="popup-active" class="menu-nav-link">
+              <img
+                class="navigation-icon"
+                src="./content/popup-button.png"
+                alt="popup button"
```
Comment (A-Ostrovnyy) [inline submissions/mimoauto/html-css-popup/index.html:38]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/mimoauto/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                id="more-active"
+                class="nav-popup-more-check-input"
+                type="checkbox"
+              />
+              <label
+                id="more-active-label"
+                for="more-active"
+                class="nav-popup-more-check"
+              >
+                <span class="popup-more-lable">More</span>
```
Comment (A-Ostrovnyy) [inline submissions/mimoauto/html-css-popup/index.html:120]: Add styles on focus state

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/mimoauto/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Roboto, Helvetica, Arial, sans-serif;
+    --logo-color: #2196F3;
+    --main-text-color: rgb(103, 103, 103);
+    --premain-font-weight: 700;
+    --logo-font-family: 'Raleway',
+        sans-serif;
```
[the remark this answers is not in the document]
Comment (mimoauto) [reply submissions/mimoauto/html-css-popup/style.css:11]: 🔥

File: submissions/mimoauto/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+.menu-nav-popup-item-link {
+    display: flex;
+    justify-content: center;
+    height: 66px;
+    border-radius: 3px;
+    transition: ease-in-out 0.3s;
+}
+
+.menu-nav-popup-item a,
```
[the remark this answers is not in the document]
Comment (mimoauto) [reply submissions/mimoauto/html-css-popup/style.css:160]: Got it! Will fix it

File: submissions/mimoauto/html-css-popup/index.html (hunk 1 above)
Comment (mimoauto) [reply submissions/mimoauto/html-css-popup/index.html]: Ok, I will leave the alt text empty

File: submissions/mimoauto/html-css-popup/index.html (hunk 2 above)
Comment (mimoauto) [reply submissions/mimoauto/html-css-popup/index.html:120]: Ok, got it

Comment (A-Ostrovnyy) [reply submissions/mimoauto/html-css-popup/index.html:120]: Stil not fixed :cry:

Comment (mimoauto) [reply submissions/mimoauto/html-css-popup/index.html:120]: Please, check again

**Review by A-Ostrovnyy — APPROVED** [review]
Awesome!

**Pull request merged**